models/topics.model.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+exports.sendTopics = () => {
```

Comment:
I'd suggest renaming this function to something  like 'fetch topics' as it should describe the interaction it's having with the database. If you consider how our model works - it goes into the database, retrieves the data we need, and then passes it up to the controller which THEN sends the information that we have selected.

Reply:
Thanks, renamed to fetchTopics